### PR TITLE
Updated various playlists

### PR DESCRIPTION
**Pull request by F640**
- Updated ch.m3u, cn.m3u, de.m3u, es.m3u, fr.m3u, hk.m3u, id.m3u, in.m3u, it.m3u, jp.m3u, kr.m3u, my.m3u, pt.m3u, sg.m3u, us.m3u, and vn_fptplay.m3u.
- Added most Dens TV URL to playlists (probably will need its dedicated playlist for easier management like Samsung, Rakuten, Fptplay, etc) along its not geo-blocked version (will keep its geo-blocked version in some cases that URL fails to connect).
- Marked CNN Indonesia as geo-blocked since it gives error 404 if played with non Indonesian IP now.
- Added backup stream for Trans TV. They have 2 streaming servers from their website.
- Added most channels from kugo app: https://app1.ninmedia.net/ninmedia-mobile-server/api/General/getStreamChannelList.json (json-encoded). It has autenthication payload, but seems not required to access it.
- Corrected Animax EPG ID.
- Edited some channels.

### CODE REVIEW

File: streams/id.m3u [hunk 1]
```diff
@@ -145,6 +159,8 @@ https://v3.siar.us/humabetangtv/live/playlist.m3u8
 http://iamchannel.org:1935/tes/1/playlist.m3u8
 #EXTINF:-1 tvg-id="Indosiar.id" status="error",Indosiar (480p) [Geo-blocked]
 http://rr2.dens.tv/s/s04/01.m3u8
+#EXTINF:-1 tvg-id="Indosiar.id" status="error",Indosiar (480p)
+http://210.210.155.37/qwr9ew/s/s04/01.m3u8
```
Comment (LaneSh4d0w) [inline streams/id.m3u:163]:
```suggestion
http://210.210.155.37/qwr9ew/s/s04/index.m3u8
```

File: streams/id.m3u [hunk 2]
```diff
@@ -244,20 +262,32 @@ http://wowza60.indostreamserver.com:1935/muadztv/live/playlist.m3u8
...
+#EXTINF:-1 tvg-id="NabawiTV.id" status="online",Nabawi TV (576p)
+https://alfa.kugo.id/nabawitv/nabawitv/playlist.m3u8
 #EXTINF:-1 tvg-id="NET.id" status="online",NET. [Not 24/7]
 http://free.fullspeed.tv/iptv-query?streaming-ip=https://youtube.com/user/NetInsights/live
 #EXTINF:-1 tvg-id="NET.id" status="online",NET. (576p)
 https://bravo.kugo.id/nettv/udp.stream/chunklist.m3u8
 #EXTINF:-1 tvg-id="NET.id" status="error",NET. (480p) [Geo-blocked]
 http://rr2.dens.tv/s/s08/01.m3u8
+#EXTINF:-1 tvg-id="NET.id" status="error",NET. (480p)
+http://210.210.155.37/qwr9ew/s/s08/01.m3u8
```
Comment (LaneSh4d0w) [inline streams/id.m3u:290]:
```suggestion
http://210.210.155.37/qwr9ew/s/s08/index.m3u8
```

File: streams/id.m3u [hunk 3]
```diff
@@ -332,6 +374,8 @@ https://bravo.kugo.id/salingsapa/rtmp.stream/playlist.m3u8
 https://cdn.gunadarma.ac.id/streams/sangajitv/ingestsangajitv.m3u8
 #EXTINF:-1 tvg-id="SCTV.id" status="error",SCTV (720p) [Geo-blocked]
 http://rr2.dens.tv/s/s03/01.m3u8
+#EXTINF:-1 tvg-id="SCTV.id" status="error",SCTV (720p)
+http://210.210.155.37/qwr9ew/s/s03/01.m3u8
```
Comment (LaneSh4d0w) [inline streams/id.m3u:378]:
```suggestion
http://210.210.155.37/qwr9ew/s/s03/index.m3u8
```

File: streams/id.m3u [hunk 4]
```diff
@@ -379,10 +427,14 @@ http://118.98.229.118:1935/live/tve.stream/playlist.m3u8
...
 #EXTINF:-1 tvg-id="TVMu.id" status="online",tvMu (720p)
 https://tvmu.siar.us/tvmu/live/playlist.m3u8
+#EXTINF:-1 tvg-id="TVMu.id" status="online",tvMu (720p)
+https://bravo.kugo.id/tvmu/udp.stream/playlist.m3u8
 #EXTINF:-1 tvg-id="TVOne.id" status="online",TVOne
 http://free.fullspeed.tv/iptv-query?streaming-ip=https://youtube.com/watch?v=K9lOPKwo44w
 #EXTINF:-1 tvg-id="TVOne.id" status="error",TVOne (480p) [Geo-blocked]
 http://rr2.dens.tv/s/s105/01.m3u8
+#EXTINF:-1 tvg-id="TVOne.id" status="error",TVOne (480p)
+http://210.210.155.37/qwr9ew/s/s105/01.m3u8
```
Comment (LaneSh4d0w) [inline streams/id.m3u:437]:
```suggestion
http://210.210.155.37/qwr9ew/s/s105/index.m3u8
```

File: streams/in.m3u [hunk 5]
```diff
@@ -636,3 +640,7 @@ https://cdn.smartstream.video/smartstream-us/rplus/rplus/playlist.m3u8
 https://satsangtv.sanskargroup.in/satsangtvlive.m3u8
 #EXTINF:-1 tvg-id="" status="error",Divya TV [Geo-blocked]
 https://sneh-z5api.herokuapp.com/?c=0-9-divyatv
+#EXTINF:-1 tvg-id="" status="error",Toonz Kids (Indonesian dub) (576p) [Geo-blocked]
+http://210.210.155.35/x6bnqe/s/s81/01.m3u8
```
Comment (LaneSh4d0w) [inline streams/in.m3u:644]:
```suggestion
http://210.210.155.35/x6bnqe/s/s81/index.m3u8
```

File: streams/in.m3u [hunk 6]
```diff
@@ -636,3 +640,7 @@ https://cdn.smartstream.video/smartstream-us/rplus/rplus/playlist.m3u8
...
+#EXTINF:-1 tvg-id="" status="error",Toonz Kids (Indonesian dub) (576p)
+http://210.210.155.37/x6bnqe/s/s81/01.m3u8
```
Comment (LaneSh4d0w) [inline streams/in.m3u:646]:
```suggestion
http://210.210.155.37/x6bnqe/s/s81/index.m3u8
```

File: streams/jp.m3u [hunk 7]
```diff
@@ -1,9 +1,8 @@
 #EXTM3U
-#EXTINF:-1 tvg-id="Animax.jp" status="online" user-agent="Mozilla/5.0 Macintosh; Intel Mac OS X 10_14_5 AppleWebKit/537.36 KHTML, like Gecko Chrome/76.0.3809.25 Safari/537.36",Animax Japan (Vietnamese Subs) (720p)
-#EXTVLCOPT:http-user-agent=Mozilla/5.0 Macintosh; Intel Mac OS X 10_14_5 AppleWebKit/537.36 KHTML, like Gecko Chrome/76.0.3809.25 Safari/537.36
-https://livecdn.fptplay.net/hda3/animaxport_hls.smil/playlist.m3u8
```
Comment (LaneSh4d0w) [inline streams/jp.m3u:4]: Link works.

**Review by BellezaEmporium — COMMENTED** [review]
Feel free to look at the set of links in this zip file below.

**Comment by LaneSh4d0w** [comment]
[dens.zip](https://github.com/iptv-org/iptv/files/8749462/dens.zip)
Feel free to look at the set of links in this zip file.

File: streams/in.m3u (hunk 6 above)
Comment (F640) [reply streams/in.m3u:646]: > http://210.210.155.37/x6bnqe/s/s81/index.m3u8

Error 404

File: streams/in.m3u (hunk 5 above)
Comment (F640) [reply streams/in.m3u:644]: Same as this comment: https://github.com/iptv-org/iptv/pull/7349#discussion_r878876097

**Comment by F640** [comment]
> Feel free to look at the set of links in this zip file below.

> [dens.zip](https://github.com/iptv-org/iptv/files/8749462/dens.zip) Feel free to look at the set of links in this zip file.

Let's see what's inside this file...

In this playlist:
All links in 210.210.155.35-qw-h.m3u died
Some links in those playlists failed to load or infinite loading

Will add:
Toonmax
http://210.210.155.35/qwr9ew/s/s50/index.m3u8

Phoenix InfoNews
http://210.210.155.35/qwr9ew/s/s21/index.m3u8


Did not add those yet (outside playlist):
Phoenix Chinese
http://210.210.155.35/uq2663/h/h157/index2.m3u8 (changing index2.m3u8 to index.m3u8 will cause infinite loading)

Wion
https://bravo.kugo.id/wion/udp.stream/chunklist.m3u8 (domestic version, probably)
http://210.210.155.35/uq2663/h/h91/index.m3u8 (international version, probably)

Motorvision (different video than existing ones)
http://210.210.155.35/uq2663/h/h09/index.m3u8

Do anyone knows which one should be put in?

Some links from: https://yamcode.com/untitled-5104

File: streams/in.m3u (hunk 6 above)
Comment (LaneSh4d0w) [reply streams/in.m3u:646]: my bad, http://210.210.155.37/x6bnqe/s/s81/index1.m3u8

Comment (LaneSh4d0w) [reply streams/in.m3u:646]: yup, in this case, add http://210.210.155.37/x6bnqe/s/s81/02.m3u8 as alt.

File: streams/in.m3u (hunk 5 above)
Comment (LaneSh4d0w) [reply streams/in.m3u:644]: in this case add http://210.210.155.35/x6bnqe/s/s81/02.m3u8 as alt.

**Comment by LaneSh4d0w** [comment]
> 

Did not know about the "index1", "index2" variants, that seem to use the mnf notation instead of specific date and time notation.

**Review by BellezaEmporium — DISMISSED** [review]

**Review by BellezaEmporium — DISMISSED** [review]

**Review by BellezaEmporium — DISMISSED** [review]

**Review by its0x08 — DISMISSED** [review]

**Review by what-a — DISMISSED** [review]

**Review by BellezaEmporium — APPROVED** [review]

**Review by BellezaEmporium — DISMISSED** [review]

**Review by what-a — DISMISSED** [review]

**Comment by F640** [comment]
@LaneSh4d0w, Sorry to ping you.

**Review by BellezaEmporium — DISMISSED** [review]

**Review by BellezaEmporium — DISMISSED** [review]

File: streams/id.m3u [hunk 8]
```diff
@@ -654,15 +732,25 @@ http://103.30.246.47:1935/tvtpi/live/playlist.m3u8
...
 #EXTINF:-1 tvg-id="" status="error",Dens ShowBiz (576p) [Geo-blocked]
 http://rr3.dens.tv/s/s35/index.m3u8
+#EXTINF:-1 tvg-id="" status="error",Dens ShowBiz (576p)
+http://210.210.155.37/x6bnqe/s/s35/index.m3u8
 #EXTINF:-1 tvg-id="" status="error",Dens Food Channel (576p) [Geo-blocked]
 http://rr3.dens.tv/s/s38/index.m3u8
+#EXTINF:-1 tvg-id="" status="error",Dens Food Channel (576p)
+http://210.210.155.37/x6bnqe/s/s38/index.m3u8
+#EXTINF:-1 tvg-id="" status="error",Dens Knowledge and Learning (576p) [Geo-blocked]
+http://rr3.dens.tv/s/s36/index.m3u8
```
Comment (freearhey) [inline streams/id.m3u:742]: image

**Review by freearhey — CHANGES_REQUESTED** [review]
Please check the marked links again.

File: streams/id.m3u [hunk 9]
```diff
@@ -314,7 +350,13 @@ https://cdn-telkomsel-01.akamaized.net/Content/HLS/Live/channel(6f7158e4-f151-45
 https://tv.rodja.live/live/ngrp:rodja_all/playlist.m3u8
 #EXTINF:-1 tvg-id="RRINet.id" status="online",RRI Net (720p) [Not 24/7]
 http://36.89.47.217:11935/rrinet/live/index.m3u8
-#EXTINF:-1 tvg-id="RuangTrampil.id" status="blocked",Ruang Trampil (480p) [Geo-blocked]
+#EXTINF:-1 tvg-id="RajawaliTV.id" status="online",RTV (480p) [Geo-blocked]
+https://cdn-telkomsel-01.akamaized.net/Content/HLS/Live/channel(41869351-4a9c-4e57-b54b-8cf5dfc919e1)/index.m3u8
```
[the remark this answers is not in the document]
Comment (F640) [reply streams/id.m3u:354]: Geo-blocked

File: streams/us.m3u [hunk 10]
```diff
@@ -130,6 +130,13 @@ https://content.uplynk.com/channel/5688add7ce704ce1a27ab62bb44044b9.m3u8
 http://appavang.flashmediacast.com:1935/Appavang/livestream/palylist.m3u8
 #EXTINF:-1 tvg-id="AWEEncore.us" status="online",AWE Encore (720p) [Geo-blocked]
 https://cdn.herringnetwork.com/80A4DFF/awee_nva/AWE_Encore.m3u8
+#EXTINF:-1 tvg-id="AXNEastAsia.us" status="error" user-agent="Mozilla/5.0 Macintosh; Intel Mac OS X 10_14_5 AppleWebKit/537.36 KHTML, like Gecko Chrome/76.0.3809.25 Safari/537.36",AXN East Asia (Thai) (720p) [Not 24/7]
+#EXTVLCOPT:http-user-agent=Mozilla/5.0 Macintosh; Intel Mac OS X 10_14_5 AppleWebKit/537.36 KHTML, like Gecko Chrome/76.0.3809.25 Safari/537.36
+https://www.livedoomovie.com/02_AXNHD_720p/chunklist.m3u8
```
[the remark this answers is not in the document]
Comment (F640) [reply streams/us.m3u:135]: I don't remember changing this

File: streams/id.m3u [hunk 11]
```diff
@@ -314,7 +350,13 @@ https://cdn-telkomsel-01.akamaized.net/Content/HLS/Live/channel(6f7158e4-f151-45
...
+#EXTINF:-1 tvg-id="RajawaliTV.id" status="online",RTV (576p) [Geo-blocked]
+http://rr1.dens.tv/h/h10/index.m3u8
```
[the remark this answers is not in the document]
Comment (F640) [reply streams/id.m3u:356]: Geo-blocked

File: streams/id.m3u [hunk 12]
```diff
@@ -368,6 +414,8 @@ http://wms.klikhost.com:1935/tegartv/tegartv/playlist.m3u8
 https://video.detik.com/trans7/smil:trans7.smil/playlist.m3u8
 #EXTINF:-1 tvg-id="TransTV.id" status="online",Trans TV (720p)
 https://video.detik.com/transtv/smil:transtv.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="TransTV.id" status="online",Trans TV (720p)
+https://video.detik.com/transtv/smil:transtv-live.smil/playlist.m3u8
```
[the remark this answers is not in the document]
Comment (F640) [reply streams/id.m3u:418]: It works from my side. Streamtest says offline, but VLC and website play it. Tried with streamtest with Singapore server and direct playback via their website and VLC using Singapore VPN.

I'm actually adding their 1st live URL server, while existing here uses 2nd live URL server.

URL: https://www.transtv.co.id/live

**Comment by LaneSh4d0w** [comment]
I think the AXN EA came from when I've fixed conflicts. Removing this.

**Review by freearhey — APPROVED** [review]

**Review by BellezaEmporium — APPROVED** [review]

**Pull request merged**